Description: g_mutex_new() is deprecated
 We have to use g_mutex_init() instead of g_mutex_new()
 Fix FTBFS on Ubuntu
Author: Ying-Chun Liu (PaulLiu) <paulliu@debian.org>
Bug-Debian: http://bugs.debian.org/674106
Index: clutter-gesture-0.0.2.1/clutter-gesture/clutter-gesture.c
===================================================================
--- clutter-gesture-0.0.2.1.orig/clutter-gesture/clutter-gesture.c	2012-06-12 12:32:52.000000000 +0800
+++ clutter-gesture-0.0.2.1/clutter-gesture/clutter-gesture.c	2012-06-12 13:31:12.590471628 +0800
@@ -85,6 +85,7 @@
 
 
 static GMutex *lock = NULL;
+static GMutex lock_s;
 static GSList *root_actors = NULL;
 
 static gboolean
@@ -297,7 +298,8 @@
   object_class->set_property = clutter_gesture_set_property;
   object_class->get_property = clutter_gesture_get_property;
 
-  lock = g_mutex_new();
+  g_mutex_init(&lock_s);
+  lock = &lock_s;
 
   g_type_class_add_private (klass, sizeof (ClutterGesturePrivate));
 
